docs/astro.config.ts

```diff
@@ -34,6 +34,9 @@ export default defineConfig({
   },
   trailingSlash: "ignore",
   markdown: markdownConfiguration,
+  image: {
+    service: passthroughImageService(),
```

Comment:
@PoisonPhang not sure what is the impact of this, but using `sharp` was troublesome.